File: 2015/src/test/java/Day4Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day4;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class Day4Test {

    private final Day4 day;

    public Day4Test() throws Exception {
        day = new Day4(new Aoc(), 4);
    }

    @Test
    public void testPart1Output() {
        assertEquals(117946, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(3938038, (int) day.part2());
    }
}

File: 2021/kotlin/src/test/kotlin/Day4Test.kt
import nl.sbmf21.aoc21.Aoc
import nl.sbmf21.aoc21.days.Day4
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class Day4Test {

    private var day = Day4(Aoc(), 4)

    @Test
    fun testPart1Output() = assertEquals(55770, day.part1())

    @Test
    fun testPart2Output() = assertEquals(2980, day.part2())
}
